trie/disabledTrieGoRoutinesThrottler.go

```diff
@@ -0,0 +1,27 @@
+package trie
+
+type disabledTrieGoRoutinesThrottler struct {
+}
+
+// NewDisabledTrieGoRoutinesThrottler returns a new instance of a disabledTrieGoRoutinesThrottler
+func NewDisabledTrieGoRoutinesThrottler() *disabledTrieGoRoutinesThrottler {
+	return &disabledTrieGoRoutinesThrottler{}
+}
+
+// CanProcess will always return false
+func (d *disabledTrieGoRoutinesThrottler) CanProcess() bool {
+	return false
```

Comment:
return true here?

Reply:
If I return true here the goroutines manager will open an unlimited ammount of goroutines. This component should be used if you want to process sequentially. `CanProcess` =  false means that you can not process on a goroutine